From 6d39acaa9182c070dceb554171a5cca36e19fa3e Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Tue, 8 Sep 2020 16:21:03 +0800
Subject: [PATCH] Support for Delete included .asm changing - asm file

Description: Support for Delete included .asm file in the TestPkg/FakeEmuSec/Ia32/SwitchRam.asm.

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/FakeEmuSec/Ia32/SwitchRam.asm | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/TestPkg/FakeEmuSec/Ia32/SwitchRam.asm b/TestPkg/FakeEmuSec/Ia32/SwitchRam.asm
index f90c569784..a829fbd3aa 100644
--- a/TestPkg/FakeEmuSec/Ia32/SwitchRam.asm
+++ b/TestPkg/FakeEmuSec/Ia32/SwitchRam.asm
@@ -65,10 +65,7 @@ SecSwitchStack   PROC
     ;
     ; Fixup the ebp point to permanent memory
     ;
-    mov   eax, ebp
-    sub   eax, ebx
-    add   eax, ecx
-    mov   ebp, eax                ; From now, ebp is pointed to permanent memory
+    #include "ForAddTest.asm"
 
     ;
     ; Fixup callee's ebp point for PeiDispatch
-- 
2.27.0.windows.1

